tests/unit/test_viz_sankey.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+from unittest.mock import patch
+import pandas as pd
+from dune_client.viz.graphs import create_sankey
+
+
+class TestCreateSankey(unittest.TestCase):
+
+    # Setting up a dataframe for testing
+    def setUp(self):
+        self.df = pd.DataFrame(
+            {
+                "source": [
+                    "WBTC",
+                    "USDC",
+                    "USDC",
+                    "USDC",
+                    "USDC",
+                    "COMP",
+                    "DAI",
+                    "DAI",
+                    "USDT",
+                    "WBTC",
+                    "DAI",
+                    "DAI",
+                    "USDC",
+                    "MKR",
+                    "DAI",
+                    "USDT",
+                    "UNI",
+                    "USDT",
+                    "USDT",
+                    "WBTC",
+                    "USDC",
+                ],
+                "target": [
+                    "WETH",
+                    "WBTC",
+                    "COMP",
+                    "MKR",
+                    "DAI",
+                    "WETH",
+                    "COMP",
+                    "WETH",
+                    "MKR",
+                    "USDT",
+                    "MKR",
+                    "USDT",
+                    "USDT",
+                    "WETH",
+                    "WBTC",
+                    "DAI",
+                    "WETH",
+                    "WETH",
+                    "WBTC",
+                    "DAI",
+                    "UNI",
+                ],
+                "value": [
+                    2184,
+                    2076,
+                    447,
+                    158,
+                    4294,
+                    519,
+                    72,
+                    4070,
+                    123,
+                    99,
+                    85,
+                    188,
+                    4675,
+                    352,
+                    281,
+                    230,
+                    59,
+                    4482,
+                    103,
+                    171,
+                    54,
+                ],
+            }
+        )
+
+        self.predefined_colors = {
+            "USDC": "rgb(38, 112, 196)",
+            "USDT": "rgb(0, 143, 142)",
+            "WETH": "rgb(144, 144, 144)",
+            "WBTC": "rgb(247, 150, 38)",
+            "COMP": "rgb(32, 217, 152)",
+            "DAI": "rgb(254, 175, 48)",
+            "MKR": "rgb(38, 173, 158)",
+            "UNI": "rgb(255, 21, 126)",
+        }
+
+        self.columns = {"source": "source", "target": "target", "value": "value"}
```

Comment:
Why are the columns mappings of themselves to themselves? Is there an example where they aren't just the same values mapping to themselves or does this even need to be a dict?

Reply:
I was thinking of giving folks the option to name their columns differently, which ultimately has to be mapped to be source, target, value.

Reply:
Perhaps this fact can be included somewhere in the doc strings. And also here in the test you could use targets like "renamed_column"

Reply:
got it makes sense, changed!